packages/renderer/src/lib/welcome/ProviderInstalled.svelte

```diff
@@ -121,23 +140,63 @@ onDestroy(() => {
     <p class="text-base text-gray-400">
       To start working with containers, {provider.name} needs to be initialized.
     </p>
-    <label for="toggle-{provider.internalId}" class="inline-flex relative items-center my-5 cursor-pointer">
-      <input
-        type="checkbox"
-        disabled="{initializeInProgress}"
-        bind:checked="{providerToggleValue}"
-        on:change="{() => initializeProvider()}"
-        id="toggle-{provider.internalId}"
-        class="sr-only peer" />
-      <div
-        class="w-9 h-5 peer-focus:ring-violet-800 rounded-full peer bg-zinc-400 peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-4 after:w-4 after:transition-all dark:border-gray-600 peer-checked:bg-violet-600">
+
+    <div class="m-5" class:hidden="{!initializationButtonVisible}">
+      <div class="bg-[#e5e7eb]">
+        <button
+          class="float-left bg-[var(--pf-global--primary-color--300)] hover:bg-[var(--pf-global--primary-color--200)] pt-2 pr-3 pl-3 pb-2 text-[13px] mr-px  w-[180px]"
+          on:click="{onInstallationClick}">
+          {installationOptionSelected}
+        </button>
+        <button
+          class="inline-block bg-[var(--pf-global--primary-color--300)] hover:bg-[var(--pf-global--primary-color--200)] text-[13px] pt-2 pr-3 pl-3 pb-2 w-[32px]"
+          on:click="{() => updateOptionsMenu(!installationOptionsMenuVisible)}">
+          <i class="fas fa-caret-down"></i>
+        </button>
       </div>
-      <span class="ml-3 text-sm font-medium text-gray-300">Initialize {provider.name}</span>
-    </label>
-    {#if initializeInProgress}
-      <div class="flex mt-2 flex-col">
-        <div>Initializing...Please Wait...</div>
-        <div class="my-2">
+      <div class="-z-1 min-w-[130px] m-auto bg-primary text-[13px]" class:hidden="{!installationOptionsMenuVisible}">
+        <ul class="w-full outline-none bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400">
+          <!-- svelte-ignore a11y-click-events-have-key-events -->
+          <li
+            class="p-2 {installationOptionSelected === InitializeOnlyMode
+              ? 'bg-[#ffffff33]'
+              : ''} hover:text-gray-300 hover:bg-[var(--pf-global--BackgroundColor--300)] cursor-pointer"
+            on:click="{() => {
+              installationOptionSelected = InitializeOnlyMode;
+              installationOptionsMenuVisible = false;
+            }}">
+            {InitializeOnlyMode}
+            {provider.name}
+          </li>
+          <!-- svelte-ignore a11y-click-events-have-key-events -->
```

Comment:
we shouldn't ignore the a11y stuff
if you use a on:click you can use <button> then it's handling correctly a11y